A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_LateralRevisionPage.js

```diff
@@ -90,12 +90,13 @@ class CDULateralRevisionPage {
 
         let holdCell = "";
         if (isWaypoint) {
-            holdCell = "<HOLD";
+            holdCell = "<HOLD[color]inop";
             mcdu.leftInputDelay[2] = () => {
                 return mcdu.getDelaySwitchPage();
             };
             mcdu.onLeftInput[2] = () => {
                 CDUHoldAtPage.ShowPage(mcdu, waypoint, waypointIndexFP);
```

Comment:
Shouldn't this be removed then ?

Reply:
I was thinking about this... and didn't want to disable the whole page since someone did make it at some point?
so I chose to make it accessbile but show the inop message

Reply:
I don't like that solution, either it's not implemented or it is.

Reply:
alright changed 👍